projects/htop.dev/package.yml

```diff
@@ -0,0 +1,29 @@
+distributable:
+  # url: https://github.com/htop-dev/htop/archive/refs/tags/{{version}}.tar.gz
+  url: https://github.com/htop-dev/htop/releases/download/{{version}}/htop-{{version}}.tar.xz
+  strip-components: 1
+
+versions:
+  github: htop-dev/htop/tags
+  strip: /v/
+
+dependencies:
+  invisible-island.net/ncurses: 6
+
+build:
+  dependencies:
+    tea.xyz/gx/cc: c99
+    tea.xyz/gx/make: '*'
+    gnu.org/autoconf: '*'
+    gnu.org/automake: '*'
+    invisible-island.net/ncurses: 6
+  script: |
+    ./autogen.sh && \
+    ./configure --prefix=/.tea/htop.dev/ && \
```

Comment:
`--prefix={{prefix}}` this is the wrong prefix

Reply:
What is the correct prefix? Their docs (https://github.com/htop-dev/htop#install) say the `make install` will install it to `/usr/local` by default if I don't change the prefix in configure... I assumed I want it installed in my `.tea` directory.